SoObjects/Appointments/SOGoCalendarComponent.m

```diff
@@ -130,6 +130,25 @@ + (SOGoWebDAVAclManager *) webdavAclManager
   return aclManager;
 }
 
+- (void) setIsNew: (BOOL) newIsNew
+{
+  // Required for protocol <SOGoComponentOccurence>
+  // but not yet implemented.  Violently warn if
+  // used.
+  NSLog (@"SOGoCalendarComponent::setIsNew is not implemented.");
+  abort();
```

Comment:
`setIsNew:` is actually implemented by the parent class `SOGoContentObject`, so simply call this:

```objc
    [super setIsNew: newIsNew];
```